com.microsoft.teams.Teams.json

```diff
@@ -0,0 +1,106 @@
+{
+  "id": "com.microsoft.teams.Teams",
```

Comment:
Is this the app-id of the app itself or did you suggest this?

Reply:
I suggest it. Couldn't find any appdata in the base `.deb`.

Reply:
com.microsoft.Teams would be a better choice.

Reply:
Okay will rename.